examples/notebooks/Brax_Experiments_with_PGPE.ipynb

```diff
@@ -0,0 +1,514 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e8c6554d-4bdb-49b9-8c2e-956bce6ddb8a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@torch.no_grad()\n",
+    "def use_policy(torch_module: nn.Module, x: Iterable, h: Optional[Iterable] = None) -> tuple:\n",
+    "    x = torch.as_tensor(np.array(x), dtype=torch.float32)\n",
+    "    if h is None:\n",
+    "        result = torch_module(x)\n",
+    "    else:\n",
+    "        result = torch_module(x, h)\n",
+    "\n",
+    "    if isinstance(result, tuple):\n",
+    "        x, h = result\n",
+    "        x = x.numpy()\n",
+    "    else:\n",
+    "        x = result.numpy()\n",
+    "        h = None\n",
+    "        \n",
+    "    return x, h"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "28c044fe-4639-411c-be78-ac09cbe5e78f",
+   "metadata": {},
+   "source": [
+    "We now initialize a new instance of our brax environment, and trigger the jit compilation on its `reset` and `step` methods."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c1251e82-1d0f-4e43-a6c5-1ec4c0275dab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "env = brax.envs.create(env_name=\"humanoid\")\n",
+    "reset = jax.jit(env.reset)\n",
+    "step = jax.jit(env.step)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "55229cc2-aad5-4c78-b095-010a538adb40",
+   "metadata": {},
+   "source": [
+    "Below we run our policy and collect the states of the episodes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "220ec837-1d1a-401e-a144-5516bdb3e493",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "seed = random.randint(0, (2 ** 32) - 1)\n",
+    "state = reset(rng=jp.random_prngkey(seed=seed))\n",
+    "\n",
+    "h = None\n",
+    "states = []\n",
+    "cumulative_reward = 0.0\n",
+    "\n",
+    "while True:\n",
+    "    action, h = use_policy(policy, state.obs, h)\n",
+    "    state = step(state, action)\n",
+    "    cumulative_reward += float(state.reward)\n",
+    "    states.append(state)\n",
+    "    if np.abs(np.array(state.done)) > 1e-4:\n",
+    "        break"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c9bbe53a-40f5-4bf5-a063-47f1d88032d6",
+   "metadata": {},
+   "source": [
+    "Length of the episode and the total reward:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "80345e9c-0694-413d-81c2-205dfacdfb5a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "len(states), cumulative_reward"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8b6ec424-e6cc-453a-93fd-eb07e44c1bd6",
+   "metadata": {},
+   "source": [
+    "Visualization of the policy:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "29b9c3aa-8068-40bc-9ec2-e5ff759ed22c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "HTML(html.render(env.sys, [state.qp for state in states]))"
```

Comment:
For me at least, this didn't work in Firefox, but did work in Chrome. Not sure if this is a general issue, but its worth investigating. If so, then we should add a comment to the top of the notebook pointing this out. It took me a few minutes to realise that this was why nothing was displaying!

Reply:
Actually, I've been doing all my tests using firefox and it has always worked for me. Thanks to you, I now know that it also works on Chrome :) I'll keep this comment of yours in mind.